docs/conf.py

```diff
@@ -268,4 +268,7 @@ def setup(app):
 
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'http://docs.python.org/': None}
+intersphinx_mapping = {
+    'http://docs.python.org/': None,
```

Comment:
It looks like we didn't have any cross-refs to the Python docs so I've removed it